code/modules/projectiles/gun_attachables.dm

```diff
@@ -1511,6 +1511,59 @@ Defined in conflicts.dm of the #defines folder.
 	pixel_shift_y = 16
 	hud_offset_mod = 10 //A sprite long enough to touch the Moon.
 
+/obj/item/attachable/m4ra_barrel
```

Comment:
Do not do this. If the only purpose of its existence is to modify the sprite, simply _change the sprite_.

Reply:
Handled on discord